"""Задание на самопроверку.
Создайте скрипт, который будет в input() принимать строки, их необходимо будет
конвертировать в числа, добавьте try-except, чтобы строки могли быть сконвертированы в числа.
В случае удачного выполнения скрипта пусть выведется: «Вы ввели правильное число».
В конце скрипта обязательно должна быть надпись «Выход из программы».
ПРИМЕЧАНИЕ: Для отлова ошибок используйте try-except, а также блоки finally и else."""

try:
    a = int(input("Введите число"))
except ValueError as error:
    print(f"Вы ввели неправильное число, {error}")
else:
    print("Вы ввели правильное число")
finally:
    print("Выход из программы")
